packages/odyssey-react-mui/src/properties/translations/odyssey-react-mui_ok_PL.properties

```diff
@@ -0,0 +1,8 @@
+fielderror.screenreader.text = 》Éŕŕöŕ 홝한Ӝฐโ《
```

Comment:
Did we just run the script to generate these ok_PL and ok_SK files ?

Reply:
Got the context from slack. Can we please add a note in the readme somewhere that tells anyone working on the odyssey.properties file that they need to generate the ok_PL and ok_SK files manually and check them in ?

Reply:
What's that mean? I'd prefer not to have manual generation if I can avoid it. I can add this as part of the build script.

Reply:
Right now the generation of these files is not automatic. These were manually generated, since the pseudo-loc package is internal and needs VPN. If we integrated it with the build, it wont work for external developers.

Reply:
I thought grabbing all these translations was automatic as it is.

Do I need to do anything, or is this gonna be covered by you guys?

Reply:
Hi @KevinGhadyani-Okta , 
so currently, all translations EXCEPT `ok_PL` and `ok_SK` will be grabbed automatically when the english file is updated on a month-to-month basis. `ok_PL` and `ok_SK` are test languages that should be generated everytime you run build. 

For those two languages I have created the following JIRA under globalization-core: 
https://oktainc.atlassian.net/browse/OKTA-622636 (adding auto generation to odyssey)
https://oktainc.atlassian.net/browse/OKTA-622639 (investigating how we want to handle using pseudo-loc in public repos in general)

so, theoretically you guys do not need to do anything unless there is an urgency to get `OKTA-622636` done.